kuma/wiki/helpers.py

```diff
@@ -232,3 +233,24 @@ def wiki_url(path):
     Look for a wiki page in the current locale, or default to given path
     """
     return reverse('wiki.document', args=[path])
+
+
+def url_list(urls):
+    js_list = '[' + ', '.join('"%s"' % url for url in urls) + ']'
+    return jinja2.Markup(js_list)
```

Comment:
Would `asset_list` be a better name? Assuming I understand correctly that it could represent a list of CSS *or* a list of JS assets.